tensorboard/plugins/profile/pod_viewer/pod_viewer_dashboard/pod-viewer-dashboard.ts

```diff
@@ -12,11 +12,25 @@ limitations under the License.
 
 namespace pod_viewer_dashboard {
 
+interface PodStatsSequence {
+  podStatsMap: any,
+}[];
+
+interface PodViewerInputData {
+  podStatsSequence: PodStatsSequence,
+  runEnvironment?: any,
+}
+
 Polymer({
   is: 'pod-viewer-dashboard',
   properties: {
+    /**
+     * @type {?PodViewerInputData}
+     */
     data: {
       type: Object,
+      value: () => {},
```

Comment:
This would have a default value of `undefined`. You need to return an empty object.
```ts
value: () => ({})
// or
value: () => {
  return {};
},
```

Reply:
Done.